src/Microsoft.SqlTools.ServiceLayer/Agent/Jobs/JobStepsActions.cs

```diff
@@ -4,6 +4,7 @@
 //
 
 using System;
+using Microsoft.SqlTools.ServiceLayer.Agent;
```

Comment:
we should try to keep using statements sorted (though it is common & fine to sort `System` namespaces ahead of others).